### PR TITLE
[com_fields] Error message when creating a field with a name already existing

### PR DESCRIPTION
**Pull request by stellainformatica**
### Steps to reproduce the issue
Try to create a custom field (i.e. in contacts) that has the same name of a custom field created for another extension (i.e. contents).

### Expected result
The ideal would be to let creating custom fields with the same name, if they are associated to different extensions, but, if not possible, a message should explain better why that field already exists. 
I've modified the message like this:
COM_FIELDS_ERROR_UNIQUE_NAME="Another Field has the same name (remember it may be a trashed item or it may be already present as custom field in another extension)."

### Actual result
 appears this message:
"Save failed with the following error: Another Field has the same name (remember it may be a trashed item)."

Pull Request for Issue #16070 .

### CODE REVIEW

**Comment by zero-24** [comment]
@laoneo @Bakual  is this the expected behavior? that you can only create one field with the same name over the cms?

File: administrator/language/en-GB/en-GB.com_fields.ini [hunk 1]
```diff
@@ -6,7 +6,7 @@
 COM_FIELDS="Fields"
 COM_FIELDS_BATCH_GROUP_LABEL="To Move or Copy your selection please select a group."
 COM_FIELDS_BATCH_GROUP_OPTION_NONE="- No Group -"
-COM_FIELDS_ERROR_UNIQUE_NAME="Another Field has the same name (remember it may be a trashed item)."
+COM_FIELDS_ERROR_UNIQUE_NAME="Another Field has the same name (remember it may be a trashed item or it may be already present as custom field in another extension)."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_fields.ini]: as a custom field not as custom field

**Comment by stellainformatica** [comment]
Corrected, thanks Brian

**Comment by laoneo** [comment]
I see no limitation as it will be not possible to have two fields for two extensions on one form. I would rather break up that restriction than this pr.

**Comment by Bakual** [comment]
> I see no limitation as it will be not possible to have two fields for two extensions on one form. I would rather break up that restriction than this pr.

The problem isn't the `name` of the field (which needs to be unique per form). It's the `id` of the field (which needs to be unique per page). And since you in theory can have multiple forms per page, that field name/id has to be unique sitewide.

**Comment by BertaOctech** [comment]
I have tested this item :white_check_mark: successfully on f3cc64f5e6fdde54ced6dfee6d8e73c1e79c313e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16069">issues.joomla.org/tracker/joomla-cms/16069</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on f3cc64f5e6fdde54ced6dfee6d8e73c1e79c313e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16069">issues.joomla.org/tracker/joomla-cms/16069</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Pull request merged**